templates/shortcode-receipt.php

```diff
@@ -23,6 +23,8 @@
 $email        = give_get_payment_user_email( $payment->ID );
 $status       = $payment->post_status;
 $status_label = give_get_payment_status( $payment, true );
+$first_name = ! empty( $user ) && isset( $user['first_name'] ) ? $user['first_name'] : '';
```

Comment:
@dixitadusara I am seeing code formatting issue here.
Please use WordPress coding standard for code formatting.
https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards